### PR TITLE
SPCTR-2299-code-refactor

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
Removal of get_fallback from all blocks

### Screenshots
post-timeline: https://d.pr/v/pf8uou
restaurant-menu: https://d.pr/v/xQOpEi
social-share: https://d.pr/v/hKsxPD
star-rating: https://d.pr/v/UOAgHV
table-of-contents: https://d.pr/v/p7BABy
tabs: https://d.pr/v/OXDVNr

### How has this been tested?
- have added null value to the attributes in editor.
- using x-debug have checked the value of the attributes it take default values if we give null.
- by giving a value to the attributes it take the current value we have passed.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: includes/blocks/star-rating/frontend.css.php [hunk 1]
```diff
@@ -10,12 +10,10 @@
...
-$block_name = 'star-rating';
-
-$rating_fallback    = UAGB_Block_Helper::get_fallback_number( $attr['rating'], 'rating', $block_name );
-$title_gap_fallback = UAGB_Block_Helper::get_fallback_number( $attr['titleGap'], 'titleGap', $block_name );
-$size_fallback      = UAGB_Block_Helper::get_fallback_number( $attr['size'], 'size', $block_name );
-$gap_fallback       = UAGB_Block_Helper::get_fallback_number( $attr['gap'], 'gap', $block_name );
+$rating_fallback    = $attr['rating'];
+$title_gap_fallback = $attr['titleGap'];
+$size_fallback      = $attr['size'];
+$gap_fallback       = $attr['gap'];
```
Comment (bsf-zanev) [inline includes/blocks/star-rating/frontend.css.php:16]: We don't require these variables.

File: includes/blocks/table-of-contents/frontend.css.php [hunk 2]
```diff
@@ -10,12 +10,10 @@
...
-$block_name = 'table-of-contents';
-
-$t_columns_desktop_fallback = UAGB_Block_Helper::get_fallback_number( $attr['tColumnsDesktop'], 'tColumnsDesktop', $block_name );
-$t_columns_tablet_fallback  = UAGB_Block_Helper::get_fallback_number( $attr['tColumnsTablet'], 'tColumnsTablet', $block_name );
-$t_columns_mobile_fallback  = UAGB_Block_Helper::get_fallback_number( $attr['tColumnsMobile'], 'tColumnsMobile', $block_name );
-$width_desktop_fallback     = UAGB_Block_Helper::get_fallback_number( $attr['widthDesktop'], 'widthDesktop', $block_name );
+$t_columns_desktop_fallback = $attr['tColumnsDesktop'];
+$t_columns_tablet_fallback  = $attr['tColumnsTablet'];
+$t_columns_mobile_fallback  = $attr['tColumnsMobile'];
+$width_desktop_fallback     = $attr['widthDesktop'];
```
Comment (bsf-zanev) [inline includes/blocks/table-of-contents/frontend.css.php:16]: We don't require these variables either.

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Please address the comments below.

File: includes/blocks/star-rating/frontend.css.php (hunk 1 above)
Comment (rajmohanreddy774) [reply includes/blocks/star-rating/frontend.css.php]: removed

File: includes/blocks/table-of-contents/frontend.css.php (hunk 2 above)
Comment (rajmohanreddy774) [reply includes/blocks/table-of-contents/frontend.css.php]: removed

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by rahulvarma722 — APPROVED** [review]

**Pull request merged**